samples/bluetooth/radio_coex_3wire/boards/nrf52840dk_nrf52840.overlay

```diff
@@ -0,0 +1,17 @@
+/* Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/ {
+  radio_coex: radio_coex_three_wire {
```

Comment:
Following internal discussion, we need a pointer to this from the radio node. This is enabled by https://github.com/nrfconnect/sdk-zephyr/pull/651, so this PR should be updated to depend on that one, and this DTS overlay should be updated to use the coex property introduced there, etc.

Reply:
done

Reply:
Where?

Reply:
git push issues w/ azizah, should be resolved now